src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -180,14 +181,15 @@ public ResponseEntity<Artefact> uploadPublication(
             .expiryDate(headers.getDisplayTo())
             .build();
 
-        Artefact createdItem = publicationService
-            .createPublication(artefact, payload);
+        Artefact createdItem = publicationService.createPublication(artefact, payload);
 
         logManualUpload(publicationService.maskEmail(issuerEmail), createdItem.getArtefactId().toString());
 
         // Process the created artefact by requesting channel management to generate PDF/Excel files
         // and check/trigger subscription management, async.
-        publicationService.processCreatedPublication(createdItem);
+        if (!LocationHelper.isNoMatchLocationId(createdItem.getLocationId())) {
```

Comment:
I think we should also do this for the Multipart upload one too :) Very unlikely to happen but still potentially possible